internal/pkg/service/buffer/storage/file.go

```diff
@@ -0,0 +1,57 @@
+package storage
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/local"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/staging"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/target"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model/column"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/utctime"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+const (
+	FileTypeCSV = "csv"
+)
```

Comment:
Robim to ready pre implementaciu `parquet` v buducnosti, ak by sa to pridalo do Connection, alebo by sme sa pripajali k DB priamo. Implementovany bude teraz ale iba CSV typ suborov.